lib/mayaUsd/render/vp2RenderDelegate/mayaPrimCommon.cpp

```diff
@@ -706,8 +769,6 @@ void MayaUsdRPrim::_SyncSharedData(
         if (!usdVisibility)
             _MakeOtherReprRenderItemsInvisible(reprToken, reprs);
 
-        _SyncDisplayLayerModes(refThis);
```

Comment:
This call is not needed here since the function _SyncDisplayLayerModes is already called from _InitReprCommon